install/overlays/libvirt/kustomization.yaml

```diff
@@ -20,6 +20,9 @@ configMapGenerator:
   - LIBVIRT_URI="qemu+ssh://root@192.168.122.1/system?no_verify=1" #set
   - LIBVIRT_NET="default" # set
   - LIBVIRT_POOL="default" # set
+  #- LIBVIRT_LAUNCH_SECURITY="" #sev or s390-pv
+  #- DISABLECVM="" #Uncomment and set if you want a generic VM
```

Comment:
Ideally we should not break the current behavior which is to create a non-confidential VM by default. So here it should be set:

`DISABLECVM=true`

If users want confidential VMs then they should se to `false`.